lib/clerk.rb

```diff
@@ -50,6 +51,8 @@ def initialize
         @api_key = secret_key
       end
 
+      @publishable_key = ENV["CLERK_PUBLISHABLE_KEY"]
```

Comment:
are these cannonical ENV's documented anywhere?

Reply:
yeah looks like this is new, might make sense to update the README 🤷‍♂️

Reply:
i have updated the readme file to include this ENV in the configuration example. I have also added a section about the new mechanism we have introduced.